devenv/main.tf

```diff
@@ -354,11 +353,28 @@ resource "local_file" "e2econfprivatedns" {
   count = var.private-dns ? 1 : 0
 }
 
-resource "local_file" "private_cluster_info" {
+resource "local_file" "registryconf" {
+  content  = azurerm_container_registry.acr.login_server
+  filename = "${path.module}/state/registry.txt"
+}
+
+resource "local_file" "cluster_info" {
```

Comment:
Do you think it would make sense to fold the values of the two new local_file resources into the old e2econf?

I can see some value in keeping unrelated values in separate files, but at first glance there doesn't seem to be a conceptual difference between the various local files.

Reply:
there's no conceptual difference but it has more to do with where the information is needed, namely that the e2e conf gets JSON-marshalled into a struct so I wanted to keep that to the values that are used within the testing suite itself

Reply:
That makes sense. I would personally jam them all into one file so future consumers of the values don't need to worry about which file to read. But I'm cool either way

Reply:
plus I think the conceptual separation makes sense given that the cluster_info isn't used by the test suite but rather the scripts that deploy the test, and that the other local file is just used by private tests (pushing a commit soon that separates the tf files though and that'll demonstrate that last piece pretty clearly)